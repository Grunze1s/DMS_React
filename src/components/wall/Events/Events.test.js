import React from 'react';
import Events from './index';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < Events {...props}  />)
    )
}

describe('<Events />', () => {

const dummy_data= {
    eventList:[{id:3},{id:1},{id:2},{id:4}]
}

it("Events render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

