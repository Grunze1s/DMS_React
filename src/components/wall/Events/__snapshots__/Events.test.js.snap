// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Events /> Events render 1`] = `ShallowWrapper {}`;
